### PR TITLE
feat:#508 updated orders list table and util functions

### PR DESCRIPTION
**Pull request by Yingxue0323**

### CODE REVIEW

File: src/app/orders/page.tsx [hunk 1]
```diff
@@ -1,69 +1,191 @@
...
+        </div>
       </div>
     </section>
   </div>
   );
 };
 
-export default orderList;
\ No newline at end of file
+export default StrategyDetails;
```
Comment (HuijunDev) [inline src/app/orders/page.tsx:191]: export default OrderList?

File: src/app/strategies/page.tsx [hunk 2]
```diff
@@ -11,7 +11,6 @@ import no_record from '../../assets/images/no_record.png';
 import '../styles/strategies.css';
 
 const strayegyList: React.FC = () => {
```
Comment (HuijunDev) [inline src/app/strategies/page.tsx:13]: The first letter of the component name needs to be capitalized.

**Comment by HuijunDev** [comment]
Might need to change the location of the CSS file.

**Comment by HuijunDev** [comment]
English is preferred in this project. Content in different languages should be placed in i18n. Additionally, calculations should differ for each language version.

File: src/app/strategies/page.tsx (hunk 2 above)
Comment (Yingxue0323) [reply src/app/strategies/page.tsx:13]: done

File: src/app/orders/page.tsx (hunk 1 above)
Comment (Yingxue0323) [reply src/app/orders/page.tsx:191]: export StrategyDetails,because the orderlist is a part of strategy details

**Comment by Yingxue0323** [comment]
> ent in different languages should be placed in i18n. Additionally, calculations should differ for each language version.

Yes, I will update it once Ryan's i18n branch is merged.

**Comment by Aria622** [comment]
Looks all good.

**Review by ucfyao — APPROVED** [review]

**Pull request merged**